### PR TITLE
Enable-WriteCore-feature-for-resources

### PR DESCRIPTION
**Pull request by welovej**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/resources/Azure.ResourceManager.Resources/CHANGELOG.md [hunk 1]
```diff
@@ -5,94 +5,54 @@
 ### Features Added
 
 - Added `DataBoundary` support.
-
```
Comment (ArthurMa1978) [inline sdk/resources/Azure.ResourceManager.Resources/CHANGELOG.md:8]: dont remove these lines

File: sdk/resources/Azure.ResourceManager.Resources/CHANGELOG.md [hunk 2]
```diff
@@ -5,94 +5,54 @@
...
 ## 1.1.0 (2022-06-08)
-
 ### Features Added
-
 - Add Update methods in resource classes.
-
 ## 1.0.0 (2022-04-07)
 This package is the first stable release of the Azure Resources management library.
+- Exposed `JsonModelWriteCore` for model serialization procedure.
```
Comment (ArthurMa1978) [inline sdk/resources/Azure.ResourceManager.Resources/CHANGELOG.md:55]: should add to the latest version

File: sdk/resources/Azure.ResourceManager.Resources/CHANGELOG.md (hunk 1 above)
Comment (welovej) [reply sdk/resources/Azure.ResourceManager.Resources/CHANGELOG.md]: Done

File: sdk/resources/Azure.ResourceManager.Resources/CHANGELOG.md (hunk 2 above)
Comment (welovej) [reply sdk/resources/Azure.ResourceManager.Resources/CHANGELOG.md]: Done

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**